dbt/models/rpie/schema.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+
+
+models:
+  - name: pin_codes
+  - name: pin_codes_dummy
```

Comment:
Same here: Is this "dummy" table necessary to track in the DAG?

Reply:
Would need to confirm with @wrridgeway, but I'm pretty confident this is in fact an important table.

Reply:
Cool, I'll leave it in unless/until Billy tells me to remove it.